service/src/test/java/bio/terra/cbas/dependencies/sam/TestSamService.java

```diff
@@ -280,12 +321,78 @@ void testHasPermissionForRequestWithNoToken() {
 
   @Test
   void testHasPermissionForInterruptingToken() {
-    setTokenValue(tokenCausingInterrupt);
+    setTokenValue(validTokenCausingAccessInterrupt);
     SamInterruptedException exception =
         assertThrows(SamInterruptedException.class, () -> samService.hasComputePermission());
     assertEquals(
         "Request interrupted while checking compute permissions on workspace from Sam",
         exception.getMessage());
     assertEquals(HttpStatus.INTERNAL_SERVER_ERROR, exception.getStatusCode());
   }
+
```

Comment:
These are the important tests for this PR - ensuring that the proper user id is getting stored in the MDC and is included in the output logs. This is true even when the user does not have permission to complete the action, and is included both in our human-readable-logging and normal logging.